prow/cmd/deck/static/script.js

```diff
@@ -763,6 +765,15 @@ function stateToAdj(state) {
     }
 }
 
+// TODO: Need a standardized way for accessing artifact upload location #8831
+function gubernatorToSpyglass(gubernatorLink, buildID) {
+    var cleanLink = gubernatorLink.replace("https://k8s-gubernator.appspot.com/build/", '')
```

Comment:
I still don't understand why gubernator is used. Can't you access artifacts directly from GCS? Also, this needs to be configurable.

Reply:
It is accessing GCS directly, but we need to know where they are in GCS, so this creates an equivalent URL with the GCS path from the gubernator link. 

See: https://github.com/kubernetes/test-infra/issues/8831 (per the TODO)